app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
```

Comment:
Kindly consider adding the `author_id (foreign_key)` to the comments and likes in this model just as you have done for the posts in order to complete the associations of the ERD diagram provided.

Eg: 

         has_many :comments, foreign_key: 'author_id'
         has_many :likes, foreign_key: 'author_id'

Reply:
done!